src/lib/y2storage/fake_device_factory.rb

```diff
@@ -769,6 +769,8 @@ def create_lvm_lv(parent, args)
       size = args["size"] || DiskSize.zero
       raise ArgumentError, "\"size\" missing for lvm_lv #{lv_name}" unless args.key?("size")
 
+      size = DiskSize.EiB(4) if size.unlimited? # bsc#1221222
```

Comment:
```suggestion
      size = DiskSize.EiB(4) if size.unlimited? # bsc#1221222, 2**62 B
```

Just making explicit the size of the number, I'm unfamiliar with such large prefixes